scripts/mbedtls_framework/psa_compliance.py

```diff
@@ -23,16 +26,20 @@
 
 PSA_ARCH_TESTS_REPO = 'https://github.com/ARM-software/psa-arch-tests.git'
 
-#pylint: disable=too-many-branches,too-many-statements,too-many-locals
+#pylint: disable=too-many-arguments,too-many-branches,too-many-statements,too-many-locals
```

Comment:
This function should arguably be split into smaller pieces. But I don't think it's that bad, and it wouldn't solve too-many-arguments which is the only one that became necessary because of my changes. So I don't plan to break it up.